src/components/pages/OnboardingCamera.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Renders the onboarding camera preview screen which
+ */
+
+import * as React from "react";
+import { View, Text, StyleSheet } from "react-native";
+import Camera from "../shared/Camera";
+
+type OnboardingCameraProps = {};
+
+export default class OnboardingCamera extends React.Component<
+  OnboardingCameraProps
+> {
+  render() {
+    return (
+      <React.Fragment>
+        <Camera
+          onVideoRecorded={uri => {
+            console.log("Video URI: " + uri);
+          }}
+        />
+        <Text style={styles.text}>
+          Example: "My name is Jane Doe and I'm joining Raha because I believe
```

Comment:
Looks like there are some whitespace issues? If you have `prettier` installed in your VSCode and autoformatting enabled it'll take care of this stuff for you automatically (in this case it'd replace the trailing space with `{" "}` since the linter prefers no trailing spaces anywhere.

Reply:
I do have prettier installed. Can you clarify which trailing space you mean? I don't think there's a trailing space anywhere. Not sure what this GitHub highlighting means.

Reply:
Looking into it, GitHub apparently thinks there's an error here based on https://github.com/github/linguist/tree/master/vendor but it's a false positive since the code runs fine